### ISSUE TITLE
Copy Params to Format/Save Params to File throws error for Multiplicity models

### ISSUE THREAD

**Issue by smk78** [SasView/sasview#2369]
**Describe the bug**
Whilst using the Unified_Power_rg model, User JamesD observed that trying to 'Copy params to Format' or 'Save Params to File' throws errors and does not work. The bug is easily to replicate.
```
15:23:15 - ERROR: Traceback (most recent call last):
  File "sas\qtgui\Perspectives\Fitting\FittingPerspective.py", line 120, in onExcelCopy
  File "sas\qtgui\Perspectives\Fitting\FittingWidget.py", line 3891, in onCopyToClipboard
  File "sas\qtgui\Perspectives\Fitting\FittingUtilities.py", line 806, in formatParametersExcel
IndexError: list index out of range
15:23:26 - ERROR: Traceback (most recent call last):
  File "sas\qtgui\Perspectives\Fitting\FittingPerspective.py", line 214, in onParamSave
  File "sas\qtgui\Perspectives\Fitting\FittingWidget.py", line 3896, in onCopyToClipboard
  File "sas\qtgui\Perspectives\Fitting\FittingUtilities.py", line 806, in formatParametersExcel
IndexError: list index out of range
```

**To Reproduce**
Run SasView and load up the Shape Independent > unified_power_Rg model
Compute the model
Attempt to Edit > Copy Params to Format/Save Params to File

**Expected behavior**
The model parameters should be exported. Though slightly different, this functionality works in 4.2.2

**Workaround**
Edit > Copy Params *does* work, but the format is not intended for the same use that Copy Params to Format/Save Params to File are

**SasView version**
Reproduced in 5.0.5, 5.0.4 & 5.0.3 so appears to be a general bug in 5.x

**Operating system (please complete the following information):**
x64/W10

**Comment by smk78** [comment]
On further testing, the same errors are thrown by the onion model, so it probably affects all multiplicity models.

**Comment by krzywon** [comment]
After some testing, the error is thrown for most/all multiplicity models. Normal parameters are lists of 7 values of the structure `['name', 'SelectedForFit', 'Value', 'Error', 'Min', 'Max', ()]` but the multiplicity values like `n` of the multi-core-shell models are lists of 3 values: `['n', 'error', 'value']`. The excel and latex export methods use indices that the multiplicity parameters don't have, causing the error.

![multiplicity_param_difference](https://user-images.githubusercontent.com/10947925/215193216-8146afb3-ee75-40c7-93d3-4a35d071f30a.png)

**Comment by lucas-wilkins** [comment]
Explicit data structures would be nice

**Comment by smk78** [comment]
Also see https://github.com/SasView/sasview/issues/1106

**Comment by butlerpd** [comment]
Should be closed by the merge of #2647. Closing manually
